gpMgmt/bin/gppylib/commands/unix.py

```diff
@@ -578,38 +579,34 @@ def get_hostname(self):
 
 
 # --------------tcp port is active -----------------------
-class PgPortIsActive(Command):
+class PgPortIsActive():
     def __init__(self, name, port, file, ctxt=LOCAL, remoteHost=None):
```

Comment:
```suggestion
    def __init__(self, port):
```

I'm surprised your IDE didn't scream at you for unused constructor parameters.

Though the bigger question is: how do we preserve the semantics when this command is called to run "remotely"? From the looks of it, we *always* probe the port locally and completely discard the remote address, am I right?